### ISSUE TITLE
Chromecast support?

### ISSUE THREAD

**Issue by tiduch** [bandesz/AirConnect-Synology#15]
Hi,

I'm trying to get the AirConnect solution working on my Synology.
It's working on my computer (Windows) but not working from your bundle or via manual instal.

I do believe the issue is firewalling rules (on top on man in de middle ;-).
I disable the one on my Syno and my router, same results and i've not great network skills.  So I'm wondering if your packages **works also for Chromecast** audio (on top of UPnP/Sonos devices) ? 

Thanks,

**Comment by mbnn** [comment]
Yes, could I pls request the addition of Aircast (so Cast devices can be seen by airplay)? :) Thx
